tasks/README.md

```diff
@@ -0,0 +1,68 @@
+# Google Cloud Tasks Pull Queue Samples
+
+Sample command-line program for interacting with the Google Cloud Tasks API
+using pull queues.
+
+Pull queues let you add tasks to a queue, then programatically remove and
+interact with them. Tasks can be added or processed in any environment,
+such as on Google App Engine or Google Compute Engine.
+
+`pull_queue_snippets.py` is a simple command-line program to demonstrate listing queues,
+ creating tasks, and pulling and acknowledging tasks.
+
+## Prerequisites to run locally:
+
+Please refer to [Setting Up a Python Development Environment](https://cloud.google.com/python/setup).
+
+## Authentication
```

Comment:
Please link to our central auth guide instead of using gcloud auth application-default login.